src/resultsbottomuppage.ui

```diff
@@ -67,7 +67,14 @@
       <item>
        <widget class="QLineEdit" name="bottomUpSearch">
         <property name="toolTip">
-         <string>Filter the call graph tree.</string>
+         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Filter the call graph tree.&lt;br/&gt;Prefix with '%' to turn it into an regex.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
@lievenhey you want to revert that line change here and in the resultstopdownpage.ui as the % is not used any more (and possibly add a note / updated screenshot for the README)

Reply:
Still included in resultstopdownpage.ui ...